index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8"/>
+        <title>CV Ekaterina Fedenko</title>
+        <link rel="stylesheet" href="./style.css"/>
+        <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+    </head>
+    <body>
+        <div class="wrap-all-content">
+            <header>
+                <h1>CV Ekaterina Fedenko</h1>
+            </header>
+            <main>
+                <div class="main-wrapper">
+                    <div class="contacts-photo-wrap">
+                        <address class="contacts">
+                            <h2>Contacts</h2><hr>
```

Comment:
By the way, ```h2``` shouldn't appear as a descendant of the ```address``` (check http://validator.w3.org/nu/?doc=https%3A%2F%2Fkatherinefedenko.github.io%2Frsschool-2019Q1-cv%2F)